partitioned-heat-conduction/dirichlet-fenics/run.sh

```diff
@@ -1,9 +1,40 @@
 #!/usr/bin/env bash
 set -e -u
 
+python3 -m venv --system-site-packages ../solver-fenics/.venv
+. ../solver-fenics/.venv/bin/activate
+pip install -r ../solver-fenics/requirements.txt
+
 . ../../tools/log.sh
 exec > >(tee --append "$LOGFILE") 2>&1
 
-python3 ../solver-fenics/heat.py Dirichlet
+if [ $# -eq 0 ] 
+then
+    echo "Running simulation with default FEniCS implementation"
+    python3 ../solver-fenics/heat.py Dirichlet
+else
+    case "$1" in
+        -[hH]|--help)
+            echo "Usage: $0 [highorder|sdc|*]"
+            echo ""
+            echo "      highorder: Run simulation with higher order FEniCS implementation"
+            echo "      sdc      : Run simulation with pySDC+FEniCS implementation"
+            echo "      *        : For every other input run the simulation with default FEniCS implementation"
```

Comment:
```suggestion
            echo "      irk : Run simulation with higher-order implicit Runge-Kutta schemes FEniCS implementation"
            echo "      sdc : Run simulation with pySDC+FEniCS implementation"
            echo "      *   : For every other input run the simulation with default FEniCS implementation"
```
`irk` instead of `highorder` because `sdc` is actually also high order. The `irk` in irksome (which was a template for Niklas' work) actually stands for Implicit Runge Kutta.